Cargo.toml

```diff
@@ -14,6 +14,7 @@ exclude = ["/ci/*", "/.github/*", "/.cirrus.yml", "/triagebot.toml"]
 description = """
 Raw FFI bindings to platform libraries like libc.
 """
+rust-version = "1.71.0"
```

Comment:
Maybe update to the 2021 edition too? The 2015 edition has quite different name resolution rules that are foreign for people having learned rust in 2018 or later as well as anyone who has forgotten the 2015 rules.

Reply:
Good point, we should migrate to it, yeah.

Reply:
But the problem is that ctest2 uses a quite old librustc_syntax fork and doesn't support newer syntax at all. We have to rewrite ctest2 first :(